src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -203,6 +214,7 @@ protected ExtensionsRunner(Extension extension) throws IOException {
         modules.add(b -> {
             b.bind(ExtensionsRunner.class).toInstance(this);
             b.bind(Extension.class).toInstance(extension);
+            b.bind(IndexNameExpressionResolver.class).toInstance(indexNameExpressionResolver);
```

Comment:
(Style Preference) I like to group things together with breaks in between them to somewhat match the same order as they are declared/assigned above. It makes maintaining code easier when you can scan down the list and insert things at the same point.  This looks odd grouped together with the two top level classes.  Even if it's in the right order it should be perhaps after that whitespace break on the line below.